lib/matplotlib/axes/_axes.py

```diff
@@ -6535,6 +6529,39 @@ def hist(self, x, bins=None, range=None, density=None, weights=None,
 
         Other Parameters
         ----------------
+        color : color or array_like of colors or None, optional
+            The meaning of the parameter depends on *histtype*:
+
+            - For filled *histtype*\s ('bar', 'barstacked' and 'stepfilled'):
+
+              The facecolor of the bars. If providing multiple datasets to
+              *x*, there must be one color per dataset. If *None* the
```

Comment:
or is it broadcasted? (dunno)

Reply:
Nope, the description is correct:

~~~
        if color is None:
            ...
        else:
            color = mcolors.to_rgba_array(color)
            if len(color) != nx:
                raise ValueError(
                    "color kwarg must have one color per data set. %d data "
~~~